src/main/java/com/foodparcel/entity/Donations.java

```diff
@@ -0,0 +1,101 @@
+package com.foodparcel.entity;
+
+import javafx.util.Builder;

```

Comment:
I think you used the wrong Builder... this is the interface from javafx.util

Reply:
deleted it! all is good :)